styles/all/template/event/topiclist_row_append.html

```diff
@@ -1,4 +1,7 @@
 <!-- IF topicrow.RH_TOPICTAGS_TAGS -->
-{topicrow.RH_TOPICTAGS_TAGS}
+<div class="rh_tags-view">{topicrow.RH_TOPICTAGS_TAGS}</div>
```

Comment:
why addubg wrappers? are they used anywere?

Reply:
hello i am not able to install on my forum 3.3.0 its showing blank page after enable all forum

Reply:
you can try again using the master branch (it supports 3.3x.x now)